<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CBCTToothSegmentationTester</class>
 <widget class="qMRMLWidget" name="CBCTToothSegmentationTester">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>1158</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton">
     <property name="text">
      <string>Inputs/Outputs</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Input volume:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="inputSelectorBox">
        <property name="toolTip">
         <string>Pick the input to the algorithm.</string>
        </property>
        <property name="nodeTypes">
         <stringlist notr="true">
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>inputVolume</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Input ROI:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="qMRMLNodeComboBox" name="inputROIBox">
        <property name="nodeTypes">
         <stringlist notr="true">
          <string>vtkMRMLMarkupsROINode</string>
         </stringlist>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="editEnabled">
         <bool>false</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
        <property name="interactionNodeSingletonTag">
         <string notr="true"/>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>inputROI</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="qMRMLMarkupsROIWidget" name="adjustROI">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Output segmentation:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="qMRMLNodeComboBox" name="outputSelectorBox">
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="toolTip">
         <string>Result with inverted threshold will be written into this volume</string>
        </property>
        <property name="nodeTypes">
         <stringlist notr="true">
          <string>vtkMRMLSegmentationNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="addEnabled">
         <bool>true</bool>
        </property>
        <property name="removeEnabled">
         <bool>true</bool>
        </property>
        <property name="editEnabled">
         <bool>false</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>outputSegmentation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="applyButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Run the algorithm.</string>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="SegmentEditorButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Segmentation Editor</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="qMRMLSegmentEditorWidget" name="SegmentEditorWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="segmentationNodeSelectorVisible">
         <bool>true</bool>
        </property>
        <property name="sourceVolumeNodeSelectorVisible">
         <bool>true</bool>
        </property>
        <property name="autoShowSourceVolumeNode">
         <bool>true</bool>
        </property>
        <property name="masterVolumeNodeSelectorVisible">
         <bool>true</bool>
        </property>
        <property name="autoShowMasterVolumeNode">
         <bool>true</bool>
        </property>
        <property name="defaultTerminologyEntrySettingsKey">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="Saveoutput">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Export to file</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="3" column="0" colspan="2">
       <widget class="qMRMLSegmentationFileExportWidget" name="FileExportWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="settingsKey">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="qMRMLNodeComboBox" name="saveSegmentation">
        <property name="nodeTypes">
         <stringlist notr="true">
          <string>vtkMRMLSegmentationNode</string>
         </stringlist>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
        <property name="interactionNodeSingletonTag">
         <string notr="true"/>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>saveSegmentation</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Segmentation:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLMarkupsROIWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLMarkupsROIWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLSegmentEditorWidget</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLSegmentEditorWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLSegmentationFileExportWidget</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLSegmentationFileExportWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>CBCTToothSegmentationTester</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputSelectorBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CBCTToothSegmentationTester</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>outputSelectorBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>8</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CBCTToothSegmentationTester</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>FileExportWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>543</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CBCTToothSegmentationTester</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputROIBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CBCTToothSegmentationTester</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>adjustROI</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CBCTToothSegmentationTester</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>saveSegmentation</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>578</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>536</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CBCTToothSegmentationTester</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>SegmentEditorWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>578</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>462</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
